node/src/test_utils/persistent_configuration_mock.rs

```diff
@@ -374,6 +388,32 @@ impl PersistentConfigurationMock {
         self
     }
 
+    pub fn min_hops_count_params(mut self, params: &Arc<Mutex<Vec<()>>>) -> Self {
```

Comment:
Shouldn't be here.

Reply:
Same as the above comment

Reply:
I don't think you were reasoning right about it.